package/yast2-firstboot.changes

```diff
@@ -1,3 +1,10 @@
+-------------------------------------------------------------------
+Thu Apr 11 09:25:14 UTC 2019 - lnussel@suse.de
+
+- Disable duplicate license dialogs (boo#131301)
```

Comment:
Note that actually is _boo#**1**131301_

https://bugzilla.suse.com/show_bug.cgi?id=1131301